MonstieWash/Assets/Scripts/MonsterAi/MonsterController.cs

```diff
@@ -121,4 +126,27 @@ public void AttackAnimationComplete()
             m_interruptedAnimation = null;
         }
     }
+
+    /// <summary>
+    /// Executes the logic required when a scene is completed (the last task is finished). Event is received from MonsterBrain <-- TaskTracker
+    /// </summary>
+    /// <param name="activeScene"> The scene that was completed. </param>
+    private void ProcessSceneComplete(Scene activeScene)
+    {
+        Debug.Log("MonsterController ProcessSceneComplete");
```

Comment:
Probably can get rid of this since we know this works and is being run.
```suggestion

```

Reply:
Whoopsies